service/controller/azurecluster/handler/azureclusteridentity/create.go

```diff
@@ -87,6 +87,12 @@ func (r *Resource) EnsureCreated(ctx context.Context, obj interface{}) error {
 			return microerror.Mask(err)
 		}
 
+		// If AzureClusterIdentity was not created by azure-operator, ignore it
+		managedBy := azureClusterIdentity.Labels[apiextensionslabels.ManagedBy]
+		if managedBy != project.Name() {
+			return nil
+		}
```

Comment:
Will managed-by label get updated to CAPZ controller during migration to CAPZ?

Reply:
I don't think so, because they won't be managed by `capz`. For `capz` clusters, `AzureClusterIdentities` and `Secrets` need to be created when creating the cluster (either the customer creates them or our UIs creates them). I tried explaining the difference scenario on the PR description.

Reply:
In that case, IIUC, we should at least remove azure-operator from managed-by label, right?

Reply:
If a customer wants to reuse the `AzureClusterIdentity` and *we remove azure-operator* then yeah, the best thing to do would be to remove the label to avoid confusion.